### PR TITLE
Ampliconsuite

### PR DESCRIPTION
**Pull request by jluebeck**
Update dependency versioning in meta.yaml  to avoid co-installation conflicts with mosek

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by jluebeck** [comment]
@BiocondaBot fetch artifacts

**Comment by jluebeck** [comment]
@BiocondaBot please fetch artifacts

**Comment by jluebeck** [comment]
@BiocondaBot please fetch artifacts

File: recipes/ampliconsuite/meta.yaml [hunk 1]
```diff
@@ -20,28 +20,26 @@ source:
 
 build:
     noarch: python
-    number: 0
+    number: 1
 
 requirements:
     build:
         - python >=3
```
Comment (bgruening) [inline recipes/ampliconsuite/meta.yaml:27]: those should be in the host section not in the build section.

File: recipes/ampliconsuite/meta.yaml [hunk 2]
```diff
@@ -20,28 +20,26 @@ source:
...
         - setuptools
-        - unzip
-        - wget
 
     run:
         - python >=3
         - bwa
         - cnvkit
         - Flask
-        - future >=0.18.3
```
Comment (bgruening) [inline recipes/ampliconsuite/meta.yaml:37]: why was it pinned in the first place? Are you sure it working with older version?

File: recipes/ampliconsuite/meta.yaml (hunk 1 above)
Comment (jluebeck) [reply recipes/ampliconsuite/meta.yaml:27]: Hi, thanks for doing this review. Can you please show me some documentation that says the build number should be placed into the host section? The documentation I see from conda shows an example where the build number is nested under the build section:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string

Comment (bgruening) [reply recipes/ampliconsuite/meta.yaml:27]: This here is not about the build number. But the dependency on python.

**Comment by jluebeck** [comment]
@BiocondaBot please fetch artifacts

**Comment by jluebeck** [comment]
@BiocondaBot please fetch artifacts

**Comment by jluebeck** [comment]
Hi @bgruening - thanks again for your review. I've re-tested the latest build on a local dataset using the artifacts above and found that it works as expected. Would you mind taking a look again when you have a moment so we can merge the PR?

**Comment by DSchreyer** [comment]
@jluebeck Looks good in terms of installation and usage. The updated bioconda ampliconsuite package was tested locally and will be integrated into the nf-core/circdna pipeline as soon as the recipe is pushed. The update removed issues with mosek which is needed for ampliconsuite and nf-core/circdna.

**Review by shenwei356 — APPROVED** [review]

**Pull request merged**